README.md

```diff
@@ -7,6 +7,8 @@ Description
 ------------
 The SMART tutorial is intended to help developers quickly create an example smart app, register it with Cerner's code console, and test it against Cerner's sandbox. It is a [Slate](https://github.com/lord/slate) site hosted through GitHub Pages and includes a small example app in the source folder.
 
+Note: This tutorial is in the process of being updated for the newer version of FHIR - R4. Outside of this tutorial, the use of R4 over DSTU2 is encouraged.
```

Comment:
Would it be helpful to include the link to our FHIR - R4 page?

Reply:
I think that's a great idea, added here: https://github.com/cerner/smart-on-fhir-tutorial/pull/168/commits/145e011ddebdf22e2f32cf3ca86313f7d3f50219

Proof of added link:
<img width="869" alt="Screen Shot 2021-08-02 at 2 05 33 PM" src="https://user-images.githubusercontent.com/40574651/127911234-741b4fc2-837b-416a-9be6-54fbc63de512.png">